src/modules/visitor-space/components/FilterMenu/FilterMenu.module.scss

```
@@ -169,4 +169,17 @@ $checkbox-size: $spacer-sm;
 	&__overlay {
 		z-index: get-z-layer("filter-overlay");
 	}
+
+	&__sorting-container {
+		margin-top: -1px;
+		padding-left: 1.7rem;
+
+		div {
+			padding: 0;
+
+			div > ul > li > div > div > button {
+				padding-left: $spacer-sm !important;
+			}
+		}
+	}
```

The selector essentially says: after the first dropdown trigger, find a menu that is open and add padding

"+" => elements that follows on the same level in the dom tree
https://developer.mozilla.org/en-US/docs/Web/CSS/Adjacent_sibling_combinator

```suggestion
	// Add padding to the sort search results dropdown
	:global(.c-dropdown__trigger:first-child + .c-dropdown__content-open) {
		:global(.c-menu) {
			padding: 0;
			margin-left: $spacer-sm;
		}
	}
```